src/libraries/System.Text.Json/src/System/Text/Json/Node/JsonNull.cs

```diff
@@ -43,23 +43,34 @@ public sealed class JsonNull : JsonNode, IEquatable<JsonNull>
         /// </summary>
         /// <param name="other">The JSON null to compare against.</param>
         /// <returns><see langword="true"/></returns>
```

Comment:
We should update the xml docs accordingly. For example:
https://github.com/dotnet/runtime/blob/3fb2aba3df7032c1267e95ca34a67d7852ed685f/src/libraries/System.Text.Json/src/System/Text/Json/Node/JsonBoolean.cs#L53-L56